src/components/Comments.js

```diff
@@ -1,21 +1,36 @@
 import React, { Component } from 'react';
 import CommentForm from './CommentForm';
-import { deleteComment } from '../redux/actions/blogActions';
+import EditComment from './EditComment';
+import { deleteComment, updateComment } from '../redux/actions/blogActions';
 import { connect } from 'react-redux';
 import '../css/comments.css';
 
 class Comments extends Component {
   state = {
-    showComments: false
+    showComments: false,
+    isEditCommentModeOn: false,
+    currentCommentId: ''
```

Comment:
Rename flag to `commentIdInEditMode`. However, the presence of this flag is actually showing the wrong abstraction level. You need a component which contains the CommentForm and the edit, delete, save, etc. buttons